From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Thu, 21 Nov 2024 16:08:43 +0100
Subject: [PATCH] Release 6.12/customers/lxa/lxatac/20241121-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 68a8faff2543..64d47e997edd 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 12
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20241121-1
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
